src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -186,6 +215,7 @@ const ImagesTable = ({ selectedBlueprint }: ImageTableProps) => {
                 isCompact
               />
             </ToolbarItem>
+            )
```

Comment:
This `)` is getting rendered to the right of the pagination :wink: 

![image](https://github.com/osbuild/image-builder-frontend/assets/95542540/08ff0743-1e4e-4691-99ed-1360e32989d2)